src/routers/download.py

```diff
@@ -1,29 +1,30 @@
-from fastapi import HTTPException, APIRouter
+import os
+
+from fastapi import APIRouter, HTTPException
 from fastapi.responses import Response
+
 from config import TMP_BASE_PATH
-import os
 
 router = APIRouter()
 
+
 @router.get("/download/{digest_id}")
-async def download_ingest(digest_id: str):
+async def download_ingest(digest_id: str) -> Response:
     try:
         # Find the first .txt file in the directory
         directory = f"{TMP_BASE_PATH}/{digest_id}"
         txt_files = [f for f in os.listdir(directory) if f.endswith('.txt')]
-        
+
         if not txt_files:
             raise FileNotFoundError("No .txt file found")
-            
-        with open(f"{directory}/{txt_files[0]}", "r") as f:
+
+        with open(f"{directory}/{txt_files[0]}") as f:
```

Comment:
Why drop the 'r' here?

Reply:
nevermind I looked and saw that it defaults to `r`

Reply:
Yeah, it was one of the hooks that removed it.